internal/cmd/egctl/testdata/config/out/out.cluster.json

```diff
@@ -0,0 +1 @@
+{"@type":"type.googleapis.com/envoy.admin.v3.ClustersConfigDump", "versionInfo":"2", "staticClusters":[{"cluster":{"@type":"type.googleapis.com/envoy.config.cluster.v3.Cluster", "name":"xds_cluster", "type":"STRICT_DNS", "connectTimeout":"1s", "loadAssignment":{"clusterName":"xds_cluster", "endpoints":[{"lbEndpoints":[{"endpoint":{"address":{"socketAddress":{"address":"envoy-gateway", "portValue":18000}}}}]}]}, "typedExtensionProtocolOptions":{"envoy.extensions.upstreams.http.v3.HttpProtocolOptions":{"@type":"type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions", "explicitHttpConfig":{"http2ProtocolOptions":{}}}}, "transportSocket":{"name":"envoy.transport_sockets.tls", "typedConfig":{"@type":"type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext", "commonTlsContext":{"tlsParams":{"tlsMaximumProtocolVersion":"TLSv1_3"}, "tlsCertificateSdsSecretConfigs":[{"name":"xds_certificate", "sdsConfig":{"pathConfigSource":{"path":"/sds/xds-certificate.json"}, "resourceApiVersion":"V3"}}], "validationContextSdsSecretConfig":{"name":"xds_trusted_ca", "sdsConfig":{"pathConfigSource":{"path":"/sds/xds-trusted-ca.json"}, "resourceApiVersion":"V3"}}}}}}, "lastUpdated":"2023-02-23T09:05:23.436Z"}], "dynamicActiveClusters":[{"versionInfo":"2a0a1698a9d3e05b802047b0cd36b52a070afa49042e1ba267168c5265c7cabf", "cluster":{"@type":"type.googleapis.com/envoy.config.cluster.v3.Cluster", "name":"default-backend-rule-0-match-0-www.example.com", "type":"STATIC", "connectTimeout":"5s", "loadAssignment":{"clusterName":"default-backend-rule-0-match-0-www.example.com", "endpoints":[{"locality":{}, "lbEndpoints":[{"endpoint":{"address":{"socketAddress":{"address":"10.96.159.190", "portValue":3000}}}, "loadBalancingWeight":1}], "loadBalancingWeight":1}]}, "dnsLookupFamily":"V4_ONLY", "outlierDetection":{}, "commonLbConfig":{"localityWeightedLbConfig":{}}}, "lastUpdated":"2023-02-23T09:05:38.443Z"}]}
```

Comment:
IIRC, default wirter is `stdout`, so maybe marshal json with indent?

Reply:
Done @zirain